tslint.json

```diff
@@ -3,7 +3,7 @@
     "tslint-defocus"
   ],
   "rulesDirectory": [
-    "codelyzer"
+    "./dist"
```

Comment:
hmm, sprawdziłeś to na projekcie używającym liba?

Reply:
nie, do sprawdzenia

Reply:
przywróciłem codelyzer

Reply:
a sprawdzisz przed merge? no bo wjedzie na mastera i nie wiemy jeszcze czy zadziała? ;-)

Reply:
sprawdzone